guardian/cq-source-snyk-full-project
# Retrieve information about snyk projects

## What does this change?

Extends the original cloudquery snyk project table to include aggregated vulnerabilities, and project tags

## How to test

Verify the plugin pulls the relevant data from snyk. Create a cloudquery spec file (the format is the same as the [existing snyk one](https://www.cloudquery.io/docs/plugins/sources/snyk/configuration)) and follow the [local development instructions](https://www.cloudquery.io/docs/developers/running-locally)

## How can we measure success?

We will be able to gain more insights from cloudquery as the tags the guardian use include the source repo. This means we can join across github data.

## Have we considered potential risks?

The risks are largely the same as the ones outlined in https://github.com/guardian/snyk-sdk-go/pull/1 , eventually I hope to merge this extension to the projects table into the main cloudquery project
